udq_actionx/UDQ_M1_RESTART.DATA

```diff
@@ -0,0 +1,430 @@
+-- ratained for back-compatibility
+-- total number of unique instances in which a UDQ is used in a keyword argument
+UDADIMS
+ 200000 1* 200000 /
+
+UDQPARAM
+/
+
+
+--PARALLEL
+-- 2 /
+
+-- *************************************************************************
+-- In this section simulation grid and static reservoir parameters are given
+-- *************************************************************************
+
+GRID 
+
+-- ****************************************************
+-------------------------------------------------------
+
+--
+--Disable echoing of the input file  
+NOECHO
+
+--
+--Requests output of an INIT file
+INIT
+
+--
+--Control output of the Grid geometry file
+GRIDFILE
+  0 1  /
+
+
+--
+--Input of pre-processor map origin (X1, Y1, X2, Y2, X3, Y3)
+--X1 Y1 The X and Y coordinates of one point of the grid Y-axis relative to the map
+--X2 Y2 The X and Y coordinates of the grid origin relative to the map origin
+--X3 Y3 The X and Y coordinates of one point of the grid X-axis relative to the map
+MAPAXES
+ 0.0 100.0 0.0 0.0 100.0 0.0  /
+--
+--
+
+NOECHO
+
+--
+-- ----------------------------------------------------
+--Include simulation grid
+INCLUDE
+  './include/test_15x10x7.grdecl' /
+
+
+PORO
+ 150*0.25
+ 150*0.20 
+ 150*0.23 
+ 150*0.18 
+ 150*0.22 
+ 150*0.21 
+ 150*0.19 
+/ 
+
+PERMX
+ 150*5000 
+ 150*1000 
+ 150*10000 
+ 150*2500 
+ 150*12500 
+ 150*1750 
+ 150*150 
+/ 
+
+-- ---------------------------------------------------
+-- Copy PERMX to PERMY  & PERMZ 
+COPY
+ PERMX   PERMY   /
+ PERMX   PERMZ   /
+/
+--
+-- ---------------------------------------------------
+-- Set Kv/Kh 
+MULTIPLY
+ PERMZ   0.1     /
+/
+
+
+
+-- ***************************************************
+-- In this section simulation grid parameters are edited
+-- ***************************************************
+
+EDIT
+
+-- ***************************************************
+
+-- ***************************************************
+-- In this section fluid-rock properties and 
+-- relative permabilities are given
+-- ***************************************************
+
+PROPS
+
+-- ***************************************************
+
+INCLUDE
+ 'include/sgof.txt' /
+
+
+INCLUDE
+ 'include/swof.txt' /
+
+-- ---------------------------------------------------
+
+-- Include PVT data
+INCLUDE
+  'include/example_pvt.txt' /
+
+-- ***********************************************************
+-- In this section simulation grid region parameters are given
+-- ***********************************************************
+
+REGIONS
+
+-- ***************************************************
+
+EQLNUM
+ 1050*1 /
+
+-- ***************************************************
+-- In this section the initialization parameters and
+-- dynamic parameters are defined
+-- ***************************************************
+
+SOLUTION
+
+RESTART
+   'UDQ_M1' 5 /
+
+
+RSVD
+ 1500 180.0
+ 4000 180.0  /
```

Comment:
This bit should technically not be in a restart case.

Reply:
OK - fixed